Resources/public/js/views/serverside/ez-roleserversideview.js

```diff
@@ -37,27 +43,134 @@ YUI.add('ez-roleserversideview', function (Y) {
         /**
          * tap event handler on the role assign buttons. It launches the
          * universal discovery widget so that the user can pick some contents.
+         * It also fill the config with datas concerning the section limitation
+         *
+         * @method _pickSubtreeWithSectionLimitation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _pickSubtreeWithSectionLimitation: function (e) {
+            var button = e.target,
+                container = this.get('container'),
+                sectionSelector = container.one(".ez-role-assignment-section-id"),
+                sectionSelectedIndex = sectionSelector.get('selectedIndex'),
+                unsetLoading = Y.bind(this._uiUnsetUDWButtonLoading, this, button),
```

Comment:
maybe also create a variable for: `sectionSelector.get('options').item(sectionSelectedIndex)`